mars/core/graph/entity.py

```diff
@@ -74,6 +84,20 @@ def results(self):
     def results(self, new_results):
         self._result_tileables = new_results
 
+    @property
+    def logic_key(self):
+        if not hasattr(self, "_logic_key") or self._logic_key is None:
+            token_keys = []
+            for node in self.topological_iter():
+                token_keys.append(
+                    tokenize(node.op.get_logic_key(), **node.extra_params)
+                )
+            self._logic_key = tokenize(*sorted(token_keys))
+            logger.debug(
+                "Got logic key: %s, token_keys: %s", self._logic_key, token_keys
```

Comment:
May just need to dump keys for tasks. Keys for all nodes bring too much information which may not be helpful for debug.